packages/provider/example/test/widget_test.dart

```diff
@@ -8,6 +8,7 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+// ignore: uri_does_not_exist
```

Comment:
I don't have this issue. Why is it needed?

Reply:
Okay, I removed it and filed as distinct issue, at https://github.com/rrousselGit/provider/issues/164.